config/locales/components/shared.yml

```diff
@@ -1,6 +1,6 @@
 en:
   personal_statement:
     title: Personal statement
-    vocation: Why do you want to become a teacher?
-    subject_knowledge: What do you know about the subject you want to teach?
+    vocation: Why do you want to be a teacher?
+    subject_knowledge: why are you suited to teach your subjects or age group?
```

Comment:
Is _why_ meant to be all lowercase here?

Reply:
no!